var/spack/repos/builtin/packages/adol-c/package.py

```diff
@@ -0,0 +1,69 @@
+from spack import *
+import sys
+
+class AdolC(Package):
+    """A package for the automatic differentiation of first and higher derivatives of vector functions in C and C++ programs by operator overloading."""
+    homepage = "https://projects.coin-or.org/ADOL-C"
+    url      = "http://www.coin-or.org/download/source/ADOL-C/ADOL-C-2.6.1.tgz"
+
+    version('head',  svn='https://projects.coin-or.org/svn/ADOL-C/trunk/')
+    version('2.6.1', '1032b28427d6e399af4610e78c0f087b')
+    
+    variant('doc',      default=True,  description='Install documentation')
+    variant('openmp',   default=False, description='Enable OpenMP support')
+    variant('sparse',   default=False, description='Enable sparse drivers')
+    variant('tests',    default=True,  description='Build all included examples as a test case')
+
+    def install(self, spec, prefix):
+        make_args = ['--prefix=%s' % prefix]
+
+        # --with-cflags=FLAGS     use CFLAGS=FLAGS (default: -O3 -Wall -ansi)
+        # --with-cxxflags=FLAGS   use CXXFLAGS=FLAGS (default: -O3 -Wall)
+
+        if '+openmp' in spec:
+            make_args.extend([
+                '--with-openmp-flag=-fopenmp' # FIXME: Is this required? -I <path to omp.h> -L <LLVM OpenMP library path>
```

Comment:
these flags are compiler specific and AFAIK they are needed in static case only, see https://github.com/davydden/spack/blob/openblas_cleanup_unit_test/var/spack/repos/builtin/packages/openblas/package.py#L73-L76

I would add another `if spec.satisfies('%gcc')` and test that is compiles with `spack install adolc%gcc+openmp`.